docs/when-to-update-my-operator.md

```diff
@@ -0,0 +1,31 @@
+# When do I need to update my operator 
+
+When you install an operator from a catalog such as [quay.io/operator-framework/upstream-community-operator](https://quay.io/repository/operator-framework/upstream-community-operators), updated versions of one of more operators are pushed to the container image by rebuilding the catalog. If you have an older image of the catalog in your cluster, you can get the updates to your operators by fetching the latest release of the catalog's container image.
+
+If the image used to build the `Catalogsource` uses a versioned tag, update the tag version of the image to fetch updates to operators in the `Catalogsource`.
+
+For example:
+
+```
+$ oc get catsrc operatorhubio-catalog -n olm -o yaml | grep image:
+    
+    image: quay.io/operator-framework/upstream-community-operators:0.0.1
+
+$ kubectl patch catsrc operatorhubio-catalog -n olm --type=merge -p '{"spec": {"image": "quay.io/operator-framework/upstream-community-operators:0.0.2"}}'
+
+```
+
+If the image used to build the `Catalogsource` uses the `latest` tag, simply delete the pod corresponding to the `CatalogSource`. When the pod is recreated, it will be recreated with the latest image of the catalog, which will contain updates to the operators in that catalog.
```

Comment:
Maybe we can also mention checking the pullimage tag is set to `always` here?

Reply:
@Bowenislandsong we omit setting the `imagePullPolicy` explicitly [here](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/controller/registry/reconciler/reconciler.go#L89), so with the `latest` tag, the `imagePullPolicy`is implicitly set to `Always`. You can refer to [this doc](https://kubernetes.io/docs/concepts/containers/images/#updating-images) for more info.

Reply:
Oh nice.